docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+version: '3'
+services:
+    
+    dalalserver:
+        container_name: dalalstreet-server
+        build: .
+        volumes: 
+            - ./dalalstreet_docker.log:/dalal-street-server/dalalstreet_docker.log
+        ports:
+            - "${SERVER_PORT}:8000"
+        depends_on:
+            - db
+        env_file:
+            - .env
+        environment:
+            MYSQL_DATABASE: ${DB_NAME}
+            DALAL_ENV: Docker
+            MYSQL_ROOT_PASSWORD: ${DB_PASS}
+            MYSQL_PORT: ${MYSQL_PORT}
+        restart: always
+
+    db:
+        container_name: dalalstreet-db
+        image: mysql:5.7
+        volumes:
+            - ./docker/mysql:/var/lib/mysql
+        env_file:
+            - .env
+        ports:
+            - "3306:3306"
+        expose:
+            - "3306" 
+        environment:
+            MYSQL_DATABASE: ${DB_NAME}
+            MYSQL_ROOT_PASSWORD: ${DB_PASS}
+    
+    phpmyadmin:
+        container_name: dalalstreet-pma
+        image: phpmyadmin/phpmyadmin
+        env_file:
+            - .env
+        ports:
+            - "${PMA_PORT}:80"
+        depends_on:
+            - db
+        # environment:
+        #     PMA_HOST: db
```

Comment:
Add this. Might be necessay in some cases. Add this in `.env` (preferred).

Reply:
Uncommenting this part causes PMA to throw a "mysqli::real_connect(): (HY000/2002): Connection refused" error. I'm not sure how to resolve it. It was uncommented in my previous commits, and worked fine. Now, even if i hard reset to those commits, it isn't working anymore. Any ideas on what the issue is?